e2e/LongHaul/device/IotHubConnectionStringHelper.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+
+namespace Microsoft.Azure.IoT.Thief.Device
+{
+    internal class IotHubConnectionStringHelper
```

Comment:
Added this class since the long-haul tests' logger needs some info from connection string provided in setting files.